include/staff/templates/status-options.tmpl.php

```diff
@@ -26,11 +26,14 @@
 
 $nextStatuses = array();
 if ($objectName == 'task') {
-    $statusName = ($object->getStatus() == 'Open') ? 'Closed' : 'Open';
-    $status = TicketStatus::objects()
-        ->filter(array('name' => $statusName))
-        ->first();
-    $nextStatuses[] = $status;
+    $state = ($object->getStatus() == 'Open') ? 'open' : 'closed';
+    foreach (TicketStatusList::getStatuses(
+                array('states' => $states)) as $status) {
+        if (isset($actions[$status->getState()])
```

Comment:
This will make sure we only show Open or Closed in the dropdown list for tasks since those are the only options you can switch to.